opm/autodiff/MissingFeatures.hpp

```diff
@@ -24,7 +24,7 @@ namespace Opm {
 
 namespace MissingFeatures {
 
-    void checkKeywords(std::shared_ptr<const Deck> deck, std::shared_ptr<const Parser> parser);
```

Comment:
> It's not a very big deal at this point, since we have not done the "big sweep of shared pointers from parser" transformation yet

I'd like to push back on this.  If we continue to consume smart pointers in our public interfaces then the conversion to a Parser module that exposes fewer smart pointers (or none at all) becomes more involved and costly because the number of places that have to change is larger.